File: PretendingNews/app/src/main/java/com/example/user/pretendingnews/SplashActivity.java
package com.example.user.pretendingnews;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

/**
 * Created by user on 2017-11-04.
 */

public class SplashActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent(SplashActivity.this, Main2Activity.class);
                SplashActivity.this.startActivity(intent);
                SplashActivity.this.finish();
            }
        },2500);
    }
}

File: PretendingNews/app/src/main/kotlin/com/example/user/pretendingnews/SplashActivity.kt
package com.example.user.pretendingnews

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import com.example.user.pretendingnews.activities.Main2Activity
import qiu.niorgai.StatusBarCompat

/**
 * Created by user on 2017-11-04.
 */

class SplashActivity : PretendingNewsBaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        StatusBarCompat.translucentStatusBar(this@SplashActivity, false)

        Handler().postDelayed({
            val intent = Intent(this@SplashActivity, MainActivity::class.java)
            intent.putExtra("query", "트럼프")
            startActivity(intent)
            finish()
        }, 2500)
    }
}
